_targets/build.project.riscv64-generic

```diff
@@ -143,6 +143,18 @@ b_image_target() {
 	size=$((size * 150 / 100))
 	echo "rootfs size: $size"
 	genext2fs -b $size -d "$PREFIX_ROOTFS" "$ROOTFS"
+
+	# Create disk image
+	PHOENIX_DISK="$PREFIX_BOOT/phoenix.disk"
+	rm -f "$PHOENIX_DISK"
+
+	# Add kernel to PHOENIX_DISK
+	printf "Copying "$PREFIX_BOOT/phoenix-kernel.img" (offs=0 blocks)\n"
+	dd if="$PREFIX_BOOT/phoenix-kernel.img" of="$PHOENIX_DISK" bs=512 conv=notrunc 2>/dev/null
+
+	# Add rootfs to PHOENIX_DISK
+	printf "Copying "$ROOTFS" (offs="$OFFS_ROOTFS" blocks)\n"
+	dd if="$ROOTFS" of="$PHOENIX_DISK" bs=512 seek="$OFFS_ROOTFS" conv=notrunc 2>/dev/null
```

Comment:
Is this seek ok? I think dd expects it in blocks, not bytes

Reply:
Yup, it's fine - we're copying 512 byte blocks starting at 2MB offset (`OFFS_ROOTFS` is given in 512-byte blocks) - it's exactly the same as on `ia32-generic` target.

Reply:
Ok, some targets (i.e zynq) define offsets within disk image in bytes, that's why I was confused